fastlane/Fastfile

```diff
@@ -756,7 +758,27 @@ platform :android do
   end
 
   def bundle_output_path(app)
-    "#{app}/build/outputs/bundle/release/#{app}-release.aab"
+    File.join(app.to_s, 'build', 'outputs', 'bundle', 'release', "#{app}-release.aab")
```

Comment:
Question (❓): Just for my FYI, why going with this change? 🤔

- Maybe because `File.join` is preferred for cross-platform compatibility, or
- Maybe because `app.to_s` converts the object stored in app to a string.

Reply:
> Maybe because File.join is preferred for cross-platform compatibility,

Yep, that's the one. And it also handles potential special characters in paths better, and it's just a better practice in Ruby in general when dealing with paths.

In practice it shouldn't make a difference since we're always building on Linux machines; but it's mostly a matter of following Ruby usual conventions and for consistency anyway.